namespaced/Detection/MobileDetect.php

```diff
@@ -17,6 +17,6 @@
  */
 
 namespace Detection;
-require_once dirname(__FILE__) . DIRECTORY_SEPARATOR . '..' . DIRECTORY_SEPARATOR . '..' . DIRECTORY_SEPARATOR . 'Mobile_Detect.php';
+require_once dirname(dirname(dirname(__FILE__))) . DIRECTORY_SEPARATOR . 'Mobile_Detect.php';
```

Comment:
@regularlabs can you please explain why this is more stable? On `2.x` I do only regex updates and bugfixing so I'm interested in not breaking anything.